src/isArray.js

```diff
@@ -0,0 +1,10 @@
+const isArray = (input) => {
+  if (Array.isArray(input)) {
```

Comment:
Also, this implementation feels a little like cheating. 😛 I wonder if we should dig into [how to tell whether or not something really is an array in javascript](http://web.mit.edu/jwalden/www/isArray.html)?